modules/nsi/NSIParser.php

```diff
@@ -127,30 +129,59 @@ function checkEncoding() {
     }
 
     function loadFile($url) {
+        Yii::trace("[NSI Parser] Loading file in URL: $url");
         $this->url = $url;
         
         $ch = curl_init();
 
         $options = array(
-                CURLOPT_RETURNTRANSFER => true,
-                CURLOPT_FOLLOWLOCATION => true,
-                CURLOPT_SSL_VERIFYHOST => false,
-                CURLOPT_SSL_VERIFYPEER => false,
-                CURLOPT_CONNECTTIMEOUT => 5,
-                CURLOPT_USERAGENT => 'Meican',
-                CURLOPT_URL => $this->url,
+            CURLOPT_RETURNTRANSFER => true,
+            CURLOPT_FOLLOWLOCATION => true,
+            CURLOPT_SSL_VERIFYHOST => false,
+            CURLOPT_SSL_VERIFYPEER => false,
```

Comment:
Should this part be false?

Reply:
Surfnet certificate is self-signed, so it won't work if we enable this flag. As it was already disabled, my choice was to keep it.